go/api/set_commands.go

```diff
@@ -278,4 +278,27 @@ type SetCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/spop/
 	SPop(key string) (Result[string], error)
+
+	// SMIsMember returns whether each member is a member of the set stored at key.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//   key - The key of the set.
+	//
+	// Return value:
+	//   A []Result[bool] containing whether each member is a member of the set stored at key.
+	//
+	// Example:
+	//	 client.SAdd("myKey", []string{"one", "two"})
```

Comment:
A silly question - is it _auto-awaited_ call?

Reply:
I assume you're asking whether this (and all other) go command method is synchronized or asynchronized. It's actually a good question, we might need to look into it.